building/tracks/syllabus/next-exercises.md

```diff
@@ -0,0 +1,27 @@
+# Syllabus - Next Exercises
+
+Having implemented your first concept exercise, the next few exercises should build on that.
+
+We like to have three to five exercises that have no other prerequisites other than the `basics` concept.
+
+Good options are the core primitives or data types of your language.
+E.g. booleans, basic numeric types, strings, dates, and atoms.
+
+Other good options are core concepts such as conditionals, loops, and basic arithmetic.
+
+It's worth checking other tracks' concept exercises to see if there are any that are appropriate for your track.
+
+Note that introducing a concept doesn't mean that you need to explain it fully in all its glory.
+You can always defer complexity.
+The language might have a dozen different numeric types that are useful in different scenarios.
+The exercise can simply mention that there are others while introducing only the most commonly used integer type and most commonly used floating-point type.
+
+Some core data types are too complex to introduce directly.
+For example strings might be lists of chars, in which case you'd want to introduce both chars and lists separately before introducing strings.
```

Comment:
I wonder whether it is easy for the reader to reconcile this part with the "hand-wave over things and explain in more detail later" text from above. I think I know what is meant here but I'm not sure everyone else does.

Reply:
That's a good call. I will elaborate a bit here to clarify.